lib/vanagon/platform/dsl.rb

```diff
@@ -51,6 +51,7 @@ def platform(name, &block)
                     end
 
         yield(self)
+        environment 'VANAGON_PLATFORM', platform_name.tr('-', '.')
```

Comment:
Well, if we use this value in a metrics aggregation system, `el.7.x86_64` would translate out as:

```
mckern@flexo Downloads $ tree -F el
el/
└── 7/
    └── x86_64/
        ├── stat_name_1
        ├── stat_name_2
        ├── stat_name_3
        ├── stat_name_4
        ├── stat_name_5
        └── stat_name_6
```